src/utils/constants.js

```diff
@@ -0,0 +1,4 @@
+export const Constants = {
```

Comment:
@ostap1010 What would be the difference between this file and the Networks.js? Why not use that one directly (or move the full Networks.js info to this one?)

Reply:
Yeah, actually that is true. I'll add "networks name" in networks.js.